### PR TITLE
[FIX] crm_iap_lead_website: correcly check for GeoIP

### PR DESCRIPTION
**Pull request by pro-odoo**
Before this commit, every time the user tried to save a rule the system displayed a message to inform that GeoIP is missing, even if GeoIP is installed.

Part of Task-ID: 2069155

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/crm_iap_lead_website/models/crm_reveal_rule.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import re
 from dateutil.relativedelta import relativedelta
```
Comment (rrahir) [inline addons/crm_iap_lead_website/models/crm_reveal_rule.py]: import odoo

File: addons/crm_iap_lead_website/models/crm_reveal_rule.py [hunk 2]
```diff
@@ -75,6 +75,7 @@ def _check_regex_url(self):
 
     @api.model
     def _assert_geoip(self):
+        from odoo import _geoip_resolver
```
Comment (rrahir) [inline addons/crm_iap_lead_website/models/crm_reveal_rule.py]: remove

File: addons/crm_iap_lead_website/models/crm_reveal_rule.py [hunk 3]
```diff
@@ -75,6 +75,7 @@ def _check_regex_url(self):
...
         if not _geoip_resolver:
```
Comment (rrahir) [inline addons/crm_iap_lead_website/models/crm_reveal_rule.py]:
```suggestion
        if not odoo._geoip_resolver:
```

**Comment by pro-odoo** [comment]
@rrahir Done, and I added another fix :)

**Comment by pro-odoo** [comment]
@robodoo r+

**Comment by pro-odoo** [comment]
@robodoo r-

**Comment by rrahir** [comment]
@robodoo  r+

**Comment by pro-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**